policyengine_us/tests/policy/baseline/gov/local/ny/nyc/tax/income/credits/school/nyc_school_tax_credit_rate_reduction_amount_eligible.yaml

```
@@ -0,0 +1,59 @@
+- name: Single person is eligible with income not over $500,000.
+  period: 2022
+  input:
+    in_nyc: true
+    nyc_school_credit_income: 500_000
```

and throughout to fix the test failures
```suggestion
    nyc_taxable_income: 500_000
```

---

though see https://github.com/PolicyEngine/policyengine-us/issues/3173#issuecomment-2049549072